Sprint-1/destructuring/exercise-3/exercise.js

```diff
@@ -6,3 +6,19 @@ let order = [
   { itemName: "Hot Coffee", quantity: 2, unitPricePence: 100 },
   { itemName: "Hash Brown", quantity: 4, unitPricePence: 40 },
 ];
+
+
+// 1. Create a function to calculate the total cost of an order
+
+
+let totalCost = 0; // declare the total cost of the order by assigning it to 0;
+
+order.forEach(item => { // implemented forEach method to iterate over the order array and calculate the total cost of the order and total.
+  const { quantity, itemName: itemName, unitPricePence } = item; // using object destruction to extract the value for each item in the order array.
+   const total = quantity * (unitPricePence / 100).toFixed(2); // we declared a variable total to calculate the quality * the unitpricepence and divided by 100 to convert pence to pounds and toFixed in two decimal place.
+
+  console.log(`${quantity} ${itemName} ${total.toFixed(2)}`); //  we used console.log to log the quantity, Name, total for each item.
```

Comment:
According to the readme file, there should be a header row in the output. Can you modify your code to output this?

![Screenshot 2025-04-20 at 9 02 49 PM](https://github.com/user-attachments/assets/f8255338-4b52-4af6-9bbb-a077fcf6a16e)